arch/arm/boot/dts/overlays/crystalfontz-cfaf7201280a0_050tx.dts

```diff
@@ -0,0 +1,119 @@
+/*
+ * crystalfontz-cfaf7201280a0_050tx.dts
```

Comment:
Please include `-overlay` in the filename, otherwise the wrong Makefile rule is used.